conf/swisnap-init.sh

```diff
@@ -25,14 +25,13 @@ swisnap_config_setup() {
     yq w -i "${PUBLISHER_PROCESSES_CONFIG}" v2.publisher.publisher-processes.all.endpoint.token -- "${SWI_TOKEN}"
 
     yq w -i ${CONFIG_FILE} log_path "${LOG_PATH:-/proc/self/fd/1}"
-    yq w -i ${CONFIG_FILE} restapi.addr 0.0.0.0
+    yq w -i ${CONFIG_FILE} restapi.addr "tcp://0.0.0.0:21413"
 
     if [ -n "${LOG_LEVEL}" ]; then
         yq w -i $CONFIG_FILE log_level "${LOG_LEVEL}"
     fi
 
     if [ "${SWISNAP_SECURE}" = true ]; then
-        yq w -i ${CONFIG_FILE} control.plugin_trust_level 1
```

Comment:
hm, now we should have SWISNAP_SECURE = false ->plugin_trust_level 0 ? Or new flag, SWISNAP_INSECURE?

Reply:
As talked it's changed now to: `if not SWISNAP_SECURE -> plugin_trust_level 0`